cmd/operator/main_test.go

```diff
@@ -32,7 +34,8 @@ func TestNamespacesType(t *testing.T) {
 	}
 
 	ns = namespaces{}
-	ns.Set(val)
+	err = ns.Set(val)
+	require.NoError(t, err)
```

Comment:
did you add this to pass golangci-lint locally?

Reply:
Linter here started complaining on that AFAIR.